src/sass/_storage.scss

```
@@ -22,6 +26,20 @@
   .p-meter {
     margin-top: 4px;
   }
+
+  @media screen and (width < 1200px) {
+    .size.clustered {
+      display: none;
+    }
+
+    .cluster-member-header {
```

```suggestion
    .cluster-member {
```
I think the `header` suffix is confusing, because in a table that is assumed to be relating to the table header. Maybe we can just remove it and naming works well. Wdyt?